### ISSUE TITLE
Support for evenodd fill rule

### ISSUE THREAD

**Issue by mitar** [Automattic/node-canvas#384]
Support [evenodd](http://www.whatwg.org/specs/web-apps/current-work/multipage/the-canvas-element.html#dom-context-2d-fillrule-evenodd) fill rule.

Example:

```
var canvas = document.createElement('canvas');
var ctx = canvas.getContext('2d');
ctx.rect(1, 1, 50, 50);
ctx.rect(5, 5, 41, 41);
ctx.fill('evenodd');
```

> The "evenodd" value indicates the even-odd rule, wherein a point is considered to be outside a shape if the number of times a half-infinite straight line drawn from that point crosses the shape's path is even.

**Comment by sdalekhin** [comment]
any news?

**Comment by isieo** [comment]
bump?

**Comment by zbjornson** [comment]
Fixed in #762/v1.4.0 🎉
